engine/engine.h

```diff
@@ -24,6 +24,12 @@
 
 namespace engine {
 
+struct Options {
+    // Default chosen by rolling 1d600.
+    int layout_width{600};
+    bool dark_mode{false};
```

Comment:
I heard light will attract bugs... maybe dark mode should be the default, and users can force enable bright mode?

Reply:
Or: this breaks my work flow, I've configured my audio player to read the average power value of each pixel on my monitor, and it detects the power spike as not using the computer and pauses my music. Just add an option to enable light mode. 

There are probably children out there using dark mode to hide from monsters! YOUR UPDATE MURDERS CHILDREN!

Reply:
So I considered the problem with the children and decided that they're holding the browser wrong and only have themselves to blame.

That's just the default if a value isn't passed on by the browser, e.g. in the tui where it doesn't matter right now.

The actual default when running the browser is trying to autodetect what the user's theme is and going with that, and then falling back to light mode if unable to detect it. I want dark mode by default, but it's not autodetected on Linux at all, so the default will be light so I'm motivated to improve this.

For now, `export HST_DARK_MODE=1` if you want to use the sane version.